utils/report/report.go

```diff
@@ -0,0 +1,97 @@
+package report
+
+import (
+	"fmt"
+	"os"
+	"text/template"
+)
+
+type Report struct {
+	ReportFile   string
+	ReportHeader string
+	ReportLines  map[ReportKey]ReportDetail
+}
+
+// Report
+type ReportKey struct {
+	Name      string
+	Cluster   string
+	Namespace string
+}
+
+type ReportDetail struct {
+	Version    string
+	ChartPath  string
+	Overrides  string
+	ValueFiles string
+}
+
+func NewReport() Report {
+
+	return Report{
+		ReportFile:   "auditreport.csv",
+		ReportHeader: "Name, Cluster, Namespace, Version, ChartPath, ValueFiles",
```

Comment:
CSV will interpret this as having a space at the start of the cell values.
```suggestion
		ReportHeader: "Name,Cluster,Namespace,Version,ChartPath,ValueFiles",
```